sanowret-crystal.lic

```diff
@@ -14,6 +14,7 @@ class SanowretCrystal
     @invalid_rooms = []
     @worn_crystal = false
     args = parse_args(arg_definitions)
+    @noun = args.sanowret_noun ? args.sanowret_noun : 'sanowret'
```

Comment:
Depending on the use case, this could be a setting in your YAML instead. How frequently/likely would this noun be to change over time?

Reply:
Yes.  That was my intention.  I'll fix this up and submit the changes.